Appraisals

```diff
@@ -89,3 +89,30 @@ appraise 'rails_6_1' do
   gem 'pg', '>= 0.18', '< 2.0'
   gem 'sqlite3', '~> 1.4'
 end
+
+appraise 'rails_7_0' do
+  instance_eval(&shared_spring_dependencies)
+  instance_eval(&shared_test_dependencies)
+
+  gem 'rails', '~> 7.0.1'
```

Comment:
Hi, @technicalpickles, hope you're doing great. Sorry for taking so long to respond. Would be possible to update Rails to 7.0.3? It seems that the new Rails version requires the most recent ruby version:https://www.fastruby.io/blog/ruby/rails/versions/compatibility-table.html.

So, we can update the ci.yml to ignore the others ruby versions like this:

```
          - { ruby: 3.0.3, appraisal: rails_7_0 }
          - { ruby: 2.7.5, appraisal: rails_7_0 }
          - { ruby: 2.6.9, appraisal: rails_7_0 }
``` 

Take a look at this commit, maybe it can help: https://github.com/VSPPedro/shoulda-matchers/pull/5/commits/b1dc1d992eb4102d2881e2da48cd5cda260fc92f